### PR TITLE
[Xedra Evolved] another batch of random stuff

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None
#### Purpose of change
Something that i made, but was stuck in giant inventor PR, so i decide to drop this at first
#### Describe the solution
Added `Debug Crafting Speed` mutation, that increases your crafting speed for 50 times
`dreamer_eff` now has correct `max_intensity`, so vitamin amount is shown properly
Added a limb score to `Blind Rage`, in future it should make the player actually almost blind
Made a invisibility artifact for eater a bit rarer
Added a new `Unbreakable` spell for eater
Fixed the description for `blood_treatment`
added `moncams` for dreamer's creatures
Removed `spell_components_constructed_hammer` requirement from dreamer's `Hammering` spell
Documented `moncams` in MUTATION.md
#### Testing
Seems to work as expected

### CODE REVIEW

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json [hunk 1]
```diff
@@ -280,5 +280,22 @@
     "color": "red",
     "scannable": false,
     "use_action": { "type": "learn_spell", "spells": [ "spell_weak" ] }
+  },
+  {
+    "id": "artifact_unbreakable",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "two spheres" },
```
Comment (GuardianDll) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json:288]:
```suggestion
    "name": { "str_sp": "two spheres" },
```

File: data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json [hunk 2]
```diff
@@ -280,5 +280,22 @@
     "color": "red",
     "scannable": false,
     "use_action": { "type": "learn_spell", "spells": [ "spell_weak" ] }
+  },
+  {
+    "id": "artifact_unbreakable",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": { "str": "pair of spheres", "str_pl": "pairs of spheres" },
+    "description": "Two perfect spheres, where the first do not move at all, and second constantly rotates around.  You feel the connection to both of them.",
```
Comment (TheShadowFerret) [inline data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_eater.json:289]:
```suggestion
    "description": "Two perfect spheres, where the first does not move at all, and second constantly rotates around.  You feel the connection to both of them.",
```

**Comment by TheShadowFerret** [comment]
While you're at it, "Banish interdimensional horrors" in eater_spells is missing `"spell_class": "EATER"` and its `energy_increment` is positive instead of negative(so it's always 1000 mana instead of lowering to 100).

File: data/mods/Xedra_Evolved/effects/effects.json [hunk 3]
```diff
@@ -117,6 +117,7 @@
       "Random voices jump into your head time to time.  Most of them are friendly.",
       "You clearly feel the otherworldly presence.  They are ready to help you."
     ],
+    "max_intensity": 5,
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/effects/effects.json:120]: Could you add a max intensity to creativity as well?

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**